### PR TITLE
[libp2p] add ListPeer rpc to return peer of topic

### PR DESCRIPTION
**Pull request by LeoHChen**
Signed-off-by: Leo Chen <leo@harmony.one>

### CODE REVIEW

**Comment by Daniel-VDM** [comment]
Could you also add this to v2 for consistency?

Can you also include a sample cURL or screenshot of Postman of RPC working so we can add to docs in the future?

File: p2p/host.go [hunk 1]
```diff
@@ -169,6 +170,17 @@ func (host *HostV2) C() (int, int, int) {
 	return len(peers), connected, not
 }
 
+// ListPeer return list of PeerID for a certain topic
+func (host *HostV2) ListPeer(topic string) []libp2p_peer.ID {
+	peers := host.joined[topic].ListPeers()
```
Comment (fxfactorial) [inline p2p/host.go:175]: sloppy. this is a race condition.

**Comment by LeoHChen** [comment]
> Could you also add this to apiv2 for consistency?
> 
> Can you also include a sample cURL or screenshot of Postman of RPC working so we can add to docs in the future?

tested on STN.

```bash
$ curl -s --request POST 'http://40.69.207.176:9500/' --header 'Content-Type: application/json' --data-raw '{ "jsonrpc": "2.0", "method": "hmy_listPeer", "params": ["hmy/stressnet/0.0.1/node/shard/1"], "id": 1}'
{"jsonrpc":"2.0","id":1,"result":["QmcuMqWUUiZLvypDvPWkj4uhz4VV67LYTxxTwCt1WwwwHd","QmfUgxd6z17ibSvpTWZnEVVVvu1xfry2Hvn89FF1VN2gy9","Qme8d5yUn95CQttsjesg5DLNkWmPFPocSxXsGnySHUk14f","QmQRHDCyxHK8aHuBsetcS2Db5JsoAafGGN1SVR8E3m5adp","QmXEt5QDBUtJPywbi3tS6SFmpkYZtfT2RauBBVgxBf7GMu","QmNeoNDpbRbPrsBcWKLFqut5HGe2vKYNBU3QudEqCstEoF","QmTxtSNbzDm1rahim5PJYgenbo7sYbXzXNpxt4LA9dF4vL","QmZUN7mZGc1PwqkJJRAdnYEoN8b2BrF7ENCpPs5jQaqV6m","QmRjDZJqk3vJj7US4x64K5NgMcojtCD6TdZ4wzHKi2SdXC","QmYLXAb8NYNSnE1gCeKp921hecpbkULhxVeJ5UER4XCi6E"]}
```

**Comment by LeoHChen** [comment]
closing this one, a new PR is proposed here. https://github.com/harmony-one/harmony/pull/3230

**Pull request closed without merging**